README.md

```diff
@@ -181,6 +181,48 @@ defineModule('plain', {
 This will result in a template file, `app/view.js` with an empty function, `function() {}`, being compiled to
 `MyApp.templates["app.view"] = function() {};`.
 
+#### options.amdName
```

Comment:
I think it makes sense to just call it `name` and then document that it only works with AMD.